FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json

```diff
@@ -92,7 +92,7 @@
         "functionAppName": "[variables('appName')]",
         "hostingPlanName": "[variables('appName')]",
         "applicationInsightsName": "[concat('AppInsights-', variables('appName'))]",
-        "synapseLinkTag": "fhir",
+        "resourceTags": {"FhirToSynapsePipeline": "FunctionV0.3.0", "FhirSchema": "R4_V1"},
```

Comment:
Nit: Maybe we can remove the `Function` in the first tag?

Reply:
FhirToCDM uses Azure DataFactory as the agent to fetch data from fhir server, just want to distinguish them in cases we may improve that.

Reply:
What about add a `_` between function and v0.3.0? like `Function_v0.3.0`

Reply:
Sure, updated